runtime/astar/src/lib.rs

```diff
@@ -1219,8 +1219,12 @@ impl_runtime_apis! {
             (weight, RuntimeBlockWeights::get().max_block)
         }
 
-        fn execute_block_no_check(block: Block) -> Weight {
-            Executive::execute_block_no_check(block)
+        fn execute_block(
+            block: Block,
+            state_root_check: bool,
+            select: frame_try_runtime::TryStateSelect
+        ) -> Weight {
```

Comment:
Whats about loggin like https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/src/lib.rs#L1970

Reply:
Sure, I will include it.

Didn't see that one since I based my update on the node template.